src/sentry_plugins/pushover/plugin.py

```diff
@@ -49,9 +55,40 @@ def get_config(self, **kwargs):
                 ],
                 'default':
                 '0',
-            }
+            }, {
+                'name':
+                'retry',
+                'label':
+                'Retry',
+                'type':
+                'number',
+                'required':
+                False,
+                'placeholder': 'e.g. 30',
+                'help': 'How often (in seconds) you will receive the same notification. Minimum of 30 seconds. Only required for "Emergency" level priority.'
+            }, {
+                'name':
+                'expire',
+                'label':
+                'Expire',
+                'type':
+                'number',
+                'required':
+                False,
+                'placeholder': 'e.g. 9000',
+                'help': 'How many seconds your notification will continue to be retried for. Maximum of 10800 seconds. Only required for "Emergency" level priority.'
+            },
         ]
 
+    def validate_config(self, project, config, actor):
+        if int(config['priority']) == 2:
+            try:
+                assert config['retry'] >= 30
```

Comment:
I see what you're doing here, and this will work totally fine. But I think a better way would be:
```python
def validate_config(self, project, config, actor):
   if int(config['priority']) == 2 and config['retry'] < 30:
     self.logger.exception(six.text_type('Retry not 30 or higher.'))
     raise PluginError('Retry must be 30 or higher.')
return config
```

Also, does `config['retry']` need to be converted at all?
And I assume you probably don't want just `Assertion error` logged.